.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,16 +15,36 @@
 ### Before
 
 
-
 ### After
 
 
+## Release checklist
+
+[Release process documentation](../doc/how-to/perform-a-release.md)
+
+As part of our continuous deployment strategy we must ensure that this work is ready to be released at any point.
+
+Before merging to `main` we must first confirm:
+
+### Pre-merge checklist
+
+- [ ] There are changes required to the Accredited Programmes API for this change to work...
+  - [ ] ... and they been released to production already
```

Comment:
This reads to me that you must confirm that changes are required to the API - perhaps `we must review the pre-merge checklist`?

Reply:
I've removed this line, I think it's fairly implicit!